fira-code-mode.el

```diff
@@ -178,6 +178,28 @@ will ensure that this range is resolved using the Fira Code Symbol font instead.
   (set-fontset-font t '(#Xe100 . #Xe16f) "Fira Code Symbol")
   (message "Finished setting up the Fira Code Symbol font."))
 
+;;;###autoload
+(defun fira-code-mode-install-fonts (&optional pfx)
+  "Helper function to download and install the latests fonts based on OS.
+When PFX is non-nil, ignore the prompt and just install"
+  (interactive "P")
+  (when (or pfx (yes-or-no-p "This will download and install fonts, are you sure you want to do this?"))
+    (let* ((font-url "https://raw.githubusercontent.com/jming422/fira-code-mode/master/fonts/FiraCode-Regular-Symbol.otf")
+           (font-dest (case window-system
```

Comment:
On my installation of Emacs, `case` is not defined - but we can just add `(require 'cl-lib)` to the top of the file and then do this (I tested that this works in `emacs -q`)
```suggestion
           (font-dest (cl-case window-system
```

Reply:
Oops, I didn't know `case` is not defined under some Emacs.
I am going to fix it and push the code again